### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by rolznz**
Add purpose and API with examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`GET /lightning-address-details?ln=<lightning_address>`
+
+### Example
+
+GET https://lnaddressproxy.getalby.com/lightning-address-details?ln=hello@getalby.com
+
+## Requesting a LNURLp invoice
+
+`POST /generate-invoice?ln=<lightning_address>&amount=<amount_in_millisates>&comment=<http_encoded_comment>`
```
Comment (im-adithya) [inline README.md]: nit: `<amount_in_millisats>`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`GET /lightning-address-details?ln=<lightning_address>`
+
+### Example
+
+GET https://lnaddressproxy.getalby.com/lightning-address-details?ln=hello@getalby.com
+
+## Requesting a LNURLp invoice
+
+`POST /generate-invoice?ln=<lightning_address>&amount=<amount_in_millisats>&comment=<http_encoded_comment>`
```
Comment (bumi) [inline README.md]:
```suggestion
`GET /generate-invoice?ln=<lightning_address>&amount=<amount_in_millisats>&comment=<http_encoded_comment>`
```
LNURL uses GET thus we use also get

in the docs we should reference also the LNURL params

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Example
+
+POST https://lnaddressproxy.getalby.com/generate-invoice?ln=hello@getalby.com&amount=1000&comment=Hello%20Alby!
```
Comment (bumi) [inline README.md]:
```suggestion
GET https://lnaddressproxy.getalby.com/generate-invoice?ln=hello@getalby.com&amount=1000&comment=Hello%20Alby!
```

File: README.md (hunk 2 above)
Comment (rolznz) [reply README.md]: @bumi  thanks, added

File: README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+### Example
+
+GET https://lnaddressproxy.getalby.com/lightning-address-details?ln=hello@getalby.com
+
+## Requesting a LNURLp invoice
+
+`GET /generate-invoice?ln=<lightning_address>&amount=<amount_in_millisats>&comment=<http_encoded_comment>`
+
+Please refer to the [https://github.com/lnurl/luds](LUDs) (LUD-06, LUD-12) etc for more information on what parameters can be passed.
```
Comment (bumi) [inline README.md]: also LUD-18 or any potental LUD

**Pull request merged**